themes/default/content/blog/pulumi-insights-ai-cli/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Pulumi Insights and AI in the Pulumi CLI"
+authors: ["luke-hoban", "kyle-pitzen"]
+meta_desc: "Pulumi Insights support for resource search and Pulumi AI is now available via the `pulumi` CLI"
+date: "2023-09-13T10:00:00-08:00"
+meta_image: "meta.png"
+tags:
+    - features
+    - ai
+    - insights
+---
+
+Earlier this year we introduced [Pulumi Insights](/blog/pulumi-insights/), a collection of features that bring intelligence to cloud infrastructure using Pulumi.  Two key components of that launch were [Pulumi AI](/blog/pulumi-ai/), a generative AI assistant purpose-built to create cloud infrastructure using natural language, and [Pulumi Resource Search](/blog/resource-search/), multi-cloud search and analytics across every cloud resource and environment in your organization.
+
+Today, we are excited to bring Pulumi Insights into the `pulumi` CLI with the new `pulumi org search` and `pulumi ai` commands. These commands put AI and resource search at your finger tips right where Pulumi users spend most of their time, in the terminal iterating on their cloud infrastructure.
+
+This release is the latest of a long list of improvements and enhancement to Pulumi Insights over the last few months, including [AI Assist for resource search](/blog/ai-assist-improvements/), [fine-grained resource property search](/blog/property-search/), [registry support in Pulumi AI](/blog/leveling-up-pulumi-ai-with-the-pulumi-registry/) and [Pulumi AI Answers](/ai/answers). We've seen incredible adoption and usage of all of these features over the last few months, and they are now easier than ever to access via the `pulumi` CLI.
+
+## Pulumi Resource Search in the CLI
+
+With the new `pulumi org search` command, you can search across all of the infrastructure managed by your organization (or individual account) using the same rich search syntax supported by Pulumi Resource Search in the Pulumi Console.
+
+```
+pulumi org search -q "type:aws:kinesis/stream:Stream modified:>=09/01/2023"
```

Comment:
[Some conversation here](https://pulumi.slack.com/archives/CCBFCGU94/p1694631147051799?thread_ts=1694586601.028689&cid=CCBFCGU94), but looks like this currently panics without a default org set, so we might want to provide a placeholder org to prevent users from hitting the panic:

```suggestion
pulumi org search -q "type:aws:kinesis/stream:Stream modified:>=09/01/2023" --org my-org
```